models/quickbooks.yml

```diff
@@ -342,10 +372,8 @@ models:
     description: >
```

Comment:
I noticed that you added unique column testing for all other end models - was there a reason the `quickbooks__balance_sheet` model doesn't have the same tests?